Bindings/Java/Matlab/Utilities/osimTableFromStruct.m

```diff
@@ -1,35 +1,13 @@
-% ----------------------------------------------------------------------- %
-% The OpenSim API is a toolkit for musculoskeletal modeling and           %
-% simulation. See http://opensim.stanford.edu and the NOTICE file         %
-% for more information. OpenSim is developed at Stanford University       %
-% and supported by the US National Institutes of Health (U54 GM072970,    %
-% R24 HD065690) and by DARPA through the Warrior Web program.             %
-%                                                                         %
-% Copyright (c) 2005-2016 Stanford University and the Authors             %
-% Author(s): James Dunne                                                  %
-%                                                                         %
-% Licensed under the Apache License, Version 2.0 (the "License");         %
-% you may not use this file except in compliance with the License.        %
-% You may obtain a copy of the License at                                 %
-% http://www.apache.org/licenses/LICENSE-2.0.                             %
-%                                                                         %
-% Unless required by applicable law or agreed to in writing, software     %
-% distributed under the License is distributed on an "AS IS" BASIS,       %
-% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
-% implied. See the License for the specific language governing            %
-% permissions and limitations under the License.                          %
-% ----------------------------------------------------------------------- %
-
-%% Convert Matlab Struct to OpenSim time Series Table
-%  Input is a Maltab stucture where data.label = nX1 or nX3 array
-%  eg structdata.LASI = [2 3 4; 4 5 6, ...
-%  One of the structures values MUST be a time vector and called 'time'
-%  Output is an OpenSim TimesSeriesTable
+function timeseriesosimtable = osimTableFromStruct(structdata)
+% Convert Matlab Struct to OpenSim time Series Table
+% Input is a Maltab stucture where data.label = nX1 or nX3 array
+% eg structdata.LASI = [2 3 4; 4 5 6, ...
```

Comment:
I would use a more concrete example with `data.<label>` and `data.time`  specified in the structure.